routes/sites.js

```diff
@@ -1,25 +1,84 @@
 'use strict';
 
-// const pool = require('../config/config').pool;
+const Promise = require('bluebird');
+const query = require('../lib/utils').query;
 
+/**
+ * Gets all sites.
+ *
+ * @param {Object} req The given request object
+ * @return {Promise} The promise
+ */
 function getSites(req) {
+  if (req.query && req.query.coach_id) {
+    return query('SELECT site_id, site_name, site_address FROM Acct NATURAL JOIN AcctToProgram NATURAL JOIN Program NATURAL JOIN Site WHERE acct_id = ' + req.query.coach_id);
+  }
 
+  return query('SELECT * FROM Site');
 }
 
+/**
+ * Creates a site.
+ *
+ * @param {Object} req The given request object
+ * @return {Promise} The promise
+ */
 function postSite(req) {
```

Comment:
This method should be named createSite(req) for consistency with the other endpoints. Don't forget to update that in the tests too! **Address before merging**